### ISSUE TITLE
Alt text not being read by aria-live region

### ISSUE THREAD

**Issue by thatjack99** [FreedomScientific/VFO-standards-support#682]
### Summary

The alt text of an image html tag is not being read in an aria-live region.

Here is one way to reproduce this:
1) Start with a div html tag with aria-live attribute
2) insert an image tag into the div with alt text. This can be done with a button
3) notice that the alt text for the image is not being read

### Expected result

I expect the alt text to be read for an image html tag in an aria-live region.

### Actual result

The alt text for an image html tag is not read in an aria-live region.

### Example
https://codepen.io/thatjack99/pen/RwJMJEM

In the example above, I expect the alt tag and div content in the aria-live region to be read after the button is clicked.
-> "this is my alt text" "this is my div message"

The actual result only reads the div content in the aria-live region after the button is clicked. 
-> "this is my div message"

#### JAWS version and build number

Jaws Version 2023.2210.29 ILM 

#### Operating System and version

Windows 10 Enterprise. 64-bit operating system, x64-based processor.
Version 21H2

#### Browser and version:

Microsoft Edge. Version 107.0.1418.56 (official build) (64-bit)

**Comment by JAWS-test** [comment]
See: https://github.com/FreedomScientific/VFO-standards-support/issues/299 for the same problem

**Comment by stevefaulkner** [comment]
tested in JAWS 2024.2312.53 can reproduce in Chrome and Firefox. alt text not announced. Works as expected in NVDA
[test case](https://codepen.io/stevef/pen/ExMXPEP)

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 68616
